compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -10281,43 +10273,8 @@ OMR::Z::CodeGenerator::setUnavailableRegisters(TR::Block *b, TR_BitVector &unava
 int8_t
 OMR::Z::CodeGenerator::getCurrentRegisterPressure(TR_RegisterKinds rk)
```

Comment:
Please remove this function as well and keep folding code that calls it assuming it always returns `0`.  It seems to have been added specifically to support restricted registers.  There are a few places in the Z evaluators that call this, so there are some folding opportunities there.